src/common/schedulers/converters.py

```diff
@@ -83,6 +83,10 @@ def from_table_to_obj_list(table, obj_type, separator="|"):
                 mapping = mappings.get(column)
                 if mapping:
                     transformation_func = mapping.get("transformation")
+                    # Adding check to avoid error for jobwatcher
+                    # cpus_per_task will be N/A if -c is not specified
+                    if mapping.get("field") == "cpus_per_task" and item == "N/A":
```

Comment:
Moving this check into `return_valid_int_value`. Keeping `cpus_per_task` because we don't have other task related CPU information. Although we are not using `cpus_per_task` at the moment, if we want to modify scaling algorithm in the future `cpus_per_task` maybe valuable.